osu.Game/Skinning/GlobalSkinnableContainerLookup.cs

```diff
@@ -31,9 +31,9 @@ public GlobalSkinnableContainerLookup(GlobalSkinnableContainers lookup, RulesetI
 
         public override string ToString()
         {
-            if (Ruleset == null) return Lookup.GetDescription();
+            if (Ruleset == null) return Lookup.GetLocalisableDescription().ToString();
```

Comment:
This will always fallback to English. Unsure how to restructure this to not use `ToString()`.

Reply:
I will harken back to my ruleset skinning configuration PR which added the target to the `ISkinComponentLookup` interface:

https://github.com/smoogipoo/osu/blob/ruleset-component-skinnin-redux/osu.Game/Skinning/ISkinComponentLookup.cs

https://github.com/smoogipoo/osu/blob/70edf15f21b3cc08668122a32ea6f49151528003/osu.Game/Overlays/SkinEditor/SkinEditor.cs#L753-L770

I still believe this is a valid path.

Reply:
I just saw this, I don't really know how this would be implemented as I don't fully understand the code flow.

I was planning to open another PR to localise the gameplay overlay (during autoplay and replays), but I will wait for this in case there's the need for `GetLocalisableDescription`.

Reply:
I think both the special dropdown path from [here](https://github.com/smoogipoo/osu/blob/70edf15f21b3cc08668122a32ea6f49151528003/osu.Game/Overlays/SkinEditor/SkinEditor.cs#L753-L770) as well as adding a name property sound fine, but maybe for simplicity let's just revert this specific change for now so that this doesn't get blocked on it?

Reply:
I'm fine with reverting the dropdown localisation for now.

Reply:
Sure.

Reply:
Ok, reverted.